client-react/src/pages/app/app-settings/ErrorPages/ErrorPageGrid.tsx

```diff
@@ -246,14 +246,15 @@ const ErrorPageGrid: React.FC<FormikProps<AppSettingsFormValues>> = props => {
   return (
     <>
       {overlay ? (
-        <div className={overlayStyle}>
+        <div className={overlayStyle} aria-label="">
```

Comment:
Avoid empty ARIA labels.
```suggestion
        <div className={overlayStyle}>
```